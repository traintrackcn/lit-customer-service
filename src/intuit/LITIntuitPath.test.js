import p from '../rPath';
import { fromJS } from 'immutable';

describe('', () => {

    it.each([
        ['p.intuit.config', p.intuit.config],
        // ['p.signUp.order.shippingMethod.id', p.signUp.order.shippingMethod.id],
    ])
    ('%s should be %s', (key, value)=>{
        expect(value).toMatchSnapshot();
    });

});