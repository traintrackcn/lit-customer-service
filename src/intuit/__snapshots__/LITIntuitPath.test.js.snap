// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`p.intuit.config should be r,intuit,config 1`] = `
Array [
  "r",
  "intuit",
  "config",
]
`;
